x-pack/test/security_solution_endpoint/services/index.ts

```diff
@@ -43,7 +43,7 @@ export const svlServices = {
   supertestWithoutAuth: KibanaSupertestWithCertWithoutAuthProvider,
 
   svlCommonApi: SvlCommonApiServiceProvider,
-  svlUserManager: SvlUserManagerProvider,
+  svlUserManager: commonFunctionalServices.samlAuthProvider,
```

Comment:
wow, this was fast - just added this line some days ago :) thank you for improving on it!